connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -46,18 +47,30 @@ object CheckConnectJvmClientCompatibility {
     sys.env("SPARK_HOME")
   }
 
+  private val sqlJar = {
```

Comment:
What is the purpose of changing the way to find jar?

Reply:
For the scala client it would pick up the assembly jar instead of the regular one. Since we only run this using SBT hardcoding it like this is fine.